esp/mlrs-wifi-bridge/mlrs-wifi-bridge-boards.h

```diff
@@ -91,14 +99,51 @@ GPIO15 = RTC_GPIO13
 //-------------------------------------------------------
 // board details
 //-------------------------------------------------------
+//-- ESP32 Dev Module
```

Comment:
should be NodeMCU ESP32-Wroom-32
dito in line 118

Reply:
Done, added also ESP32-DevKitC V4 as this runs with the same code as well.

Reply:
since it is not hardware compatible I guess a separate entry for the  ESP32-DevKitC V4 would be nicer.

you probably are going to also add the inverted version ... I'm not sure I like that huge increase of cases ...

what do you think about this:

the two cases defined MODULE_NODEMCU_ESP32_WROOM32 and defined MODULE_NODEMCU_ESP32_WROOM32_SERIAL_INVERTED could be put into one common #if, and whithin it the case MODULE_NODEMCU_ESP32_WROOM32_SERIAL_INVERTED could set the SERIAL_INVERT true. Should make the whole section much more compact.

I'm still not sure we should add the inverted variant for each and every module type coming up in life ... for the R9M we did it as a curtosy to Brad and his particular mod, which is now also in the docs, simply becasue it really fits so neatly. But to do it for each and every ?!?!?!

Maybe we rather should provide an option USE_SERIAL_INVERTED, like we do with WIFI_PROTOCOL or WIFI_POWER ... I momentarily tend towards this ...

what do you say?

Reply:
Good idea. Please have a look at latest commit.